packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -1,17 +1,20 @@
 module.exports = {
   'rules': {
     // enforce spacing inside array brackets
-    'array-bracket-spacing': 0,
+    'array-bracket-spacing': [2, 'never'],
```

Comment:
This is going to require re-linting in places that were formerly passing lint, but I like it.